client/my-sites/theme/main.jsx

```diff
@@ -243,8 +244,8 @@ const ThemeSheet = React.createClass( {
 	},
 
 	renderFeaturesCard() {
-		const themeFeatures = this.props.taxonomies && this.props.taxonomies.features instanceof Array
-		? this.props.taxonomies.features.map( function( item, i ) {
+		const themeFeatures = this.props.taxonomies && this.props.taxonomies.theme_feature instanceof Array
+		? this.props.taxonomies.theme_feature.map( function( item, i ) {
 			return ( <li key={ 'theme-features-item-' + i++ }><span>{ item.name }</span></li> );
```

Comment:
Oh BTW, do we really need to increment `i` here? Should be handled by `map`, shouldn't it?